packages/swirl-components/src/components/swirl-date-picker/swirl-date-picker.tsx

```diff
@@ -31,6 +42,17 @@ export class SwirlDatePicker {
 
   @Event({ bubbles: false }) valueChange: EventEmitter<Date | Date[]>;
 
+  componentWillLoad() {
+    if (!this.firstDayOfWeek) {
+      const locale = new Intl.Locale(this.locale) as LocaleWithWeekInfo;
+
+      this.firstDayOfWeek =
+        "getWeekInfo" in locale
+          ? locale.getWeekInfo().firstDay
+          : locale.weekInfo?.firstDay;
```

Comment:
Doesn't this mean that for firefox firstDayOfWeek will be undefined as none of those properties exist? Maybe add a final default which is 0?

Reply:
Tested it and should be fine. It defaults to 0 in https://sqrrl.github.io/wc-datepicker/

Reply:
It would be passed to `<wc-datepicker />` as  `undefined`. So it would take its default. https://github.com/Sqrrl/wc-datepicker/blob/main/src/components/wc-datepicker/wc-datepicker.tsx

Added the fallback anyway to be safe.